server/util/flagutil/types/types.go

```diff
@@ -385,10 +385,18 @@ type FlagAlias[T any] struct {
 func Alias[T any](name string, newNames ...string) *T {
 	f := &FlagAlias[T]{name: name}
 	var flg *flag.Flag
-	for aliaser, ok := common.IsNameAliasing(f), true; ok; aliaser, ok = flg.Value.(common.IsNameAliasing) {
-		if flg = common.DefaultFlagSet.Lookup(aliaser.AliasedName()); flg == nil {
+	aliaser := common.IsNameAliasing(f)
+	ok := true
```

Comment:
I'd fold the ok setting and checking into the for loop:

```
for ok := true; ok; {
   ...
}
```

Reply:
Good idea. I tried folding it to the top and ended up folding the last statement up there as well.

I think this is quite readable